metadata.json

```diff
@@ -87,6 +87,10 @@
         {
             "name": "puppetlabs/stdlib",
             "version_requirement": ">= 4.2.2 <5.0.0"
+        },
+        {
+            "name": "puppetlabs/ntp",
```

Comment:
This is only needed for acceptance testing. It feels wrong to add this as a regular dependency and should be fixed in the ```spec_helper_acceptance```.